src/playerTechs/interface.js

```diff
@@ -109,12 +109,6 @@ export class PlayerTechInterface {
     this.videoElement_.className = "eyevinn-player";
     let wrapperElement = document.getElementById(wrapperId);
     wrapperElement.appendChild(this.videoElement_);
-    let ar = this.videoElement_.clientWidth / this.videoElement_.clientHeight;
```

Comment:
@birme @slowmove 

The behavior got weird when I let the <video> fill the parent, what particular issue did this attempt to solve? Looks the same without it unless I'm missing something.

Reply:
To be honest I can't quite recall it.